plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/GroupInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
+use Automattic\WooCommerce\Admin\BlockTemplates\ContainerInterface;
+
+/**
+ * Interface for block containers.
+ */
+interface GroupInterface extends ContainerInterface {
```

Comment:
This interface should also extend `BlockInterface`. This will allow groups to be fully accessed/modified via the API (see comments in tests).

Reply:
Addressed as part of https://github.com/woocommerce/woocommerce/pull/39814/commits/dd8630fdc735e73a1fd0ab96d07f344c8f8b5e6c